### ISSUE TITLE
Ability to ignore users and supress or display alerts per-user

### ISSUE THREAD

**Issue by GoogleCodeExporter** [emuikernel/wittytwitter#134]
```
Allow users to micromanage information displayed by Witty.

Followers can be completely ignored, or alert messages can be turned off
for specified users, or be turned off by default and only turned on for
select users.
```

Original issue reported on code.google.com by `ignu.sm...@gmail.com` on 27 May 2008 at 3:19

Attachments:
- [WittyIgnoreAndAlertPreferences.patch](https://storage.googleapis.com/google-code-attachments/wittytwitter/issue-134/comment-0/WittyIgnoreAndAlertPreferences.patch)

**Comment by GoogleCodeExporter** [comment]
```
(sorry, this should've been an enhancement.  google code won't let me edit the 
issue)
```

Original comment by `ignu.sm...@gmail.com` on 27 May 2008 at 3:20

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Latest patch that works against the trunk.
```

Original comment by `ignu.sm...@gmail.com` on 5 Jan 2009 at 3:00

Attachments:
- [WittyPopUpOptions.patch](https://storage.googleapis.com/google-code-attachments/wittytwitter/issue-134/comment-6/WittyPopUpOptions.patch)

**Comment by GoogleCodeExporter** [comment]
```
Applied patch
```

Original comment by `jongallo...@gmail.com` on 5 Jan 2009 at 4:32
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
This is included in Witty 2.1
```

Original comment by `jongallo...@gmail.com` on 4 Feb 2009 at 4:37
- Changed state: **Fixed**
